src/shared/download-file.ts

```diff
@@ -49,3 +49,15 @@ export function downloadTextFile(
 ) {
   downloadFile(text, 'text/plain', filename, _document);
 }
+
+/**
+ * Download a CSV file containing text
```

Comment:
```suggestion
 * Download a CSV file.
```

Since CSV is a text format, we don't need to mention that it contains text.

Reply:
Perhaps we can skip the comment entirely then, as the function is downloadCSVFile

Reply:
True, no point in adding a comment that doesn't add value.